### PR TITLE
DOC: whatsnew for array_ufunc

### PR DESCRIPTION
**Pull request by TomAugspurger**
xref https://github.com/pandas-dev/pandas/issues/27186#issuecomment-507768331

### CODE REVIEW

File: doc/source/whatsnew/v0.25.0.rst [hunk 1]
```diff
@@ -620,6 +620,48 @@ previous behavior of returning overlapping matches.
...
+
+.. _whatsnew_0250.api_breaking.ufunc:
+
+Binary ufuncs on Series now align
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Applying a binary ufunc like :func:`numpy.power` now aligns the inputs
+when both are :class:`Series` (:issue:`23293`).
+
+.. ipython:: python:
```
Comment (jreback) [inline doc/source/whatsnew/v0.25.0.rst]: no colon after python

**Comment by jreback** [comment]
lgtm, but typo, merge on green.

File: doc/source/whatsnew/v0.25.0.rst (hunk 1 above)
Comment (TomAugspurger) [inline doc/source/whatsnew/v0.25.0.rst]:
```suggestion
.. ipython:: python
```

**Comment by jreback** [comment]
thanks @TomAugspurger

**Pull request merged**